apps/crn-server/test/helpers/retryable.ts

```diff
@@ -0,0 +1,26 @@
+type Options = {
+  retries?: number;
+  delay?: number;
+};
+
+export const retryable = async (
```

Comment:
maybe you want to use something from npm like async-retry? it's got exponential backoff, time limiting etc...and its from vercel

Reply:
I don't have any particularly strong feeling either way on this one. There's arguments in favour of doing it either way.

I've swapped out for the package you suggested, with a small wrapper to override the default retries because 10 is way more than necessary for our needs I think.